### PR TITLE
Add docs for epilogue

### PR DESCRIPTION
**Pull request by SamCarlberg**

### CODE REVIEW

File: source/docs/software/telemetry/robot-telemetry-with-annotations.rst [hunk 1]
```diff
@@ -0,0 +1,295 @@
+Robot Telemetry with Annotations (Java Only)
+============================================
+
+At its most basic, telemetry with annotations can be accomplished with only a few lines of code:
```
Comment (sciencewhiz) [inline source/docs/software/telemetry/robot-telemetry-with-annotations.rst:4]: Some type of lead-in about this being new for 2025 WPIlib I think would be helpful

File: source/docs/software/telemetry/robot-telemetry-with-annotations.rst [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+        class VendorMotor {
+          public int getFaults();
+          public void set(double speed);
+          public double get();
+          public double getAppliedVoltage();
+          public double getInputCurrent();
+        }
+
+        @CustomLoggerFor(VendorMotor.class)
+        public class YourCustomVendorMotorLogger extends ClassSpecificLogger<VendorMotor> {
```
[the remark this answers is not in the document]
Comment (SamCarlberg) [reply source/docs/software/telemetry/robot-telemetry-with-annotations.rst:268]: Yep, good idea

File: source/docs/software/telemetry/robot-telemetry-with-annotations.rst (hunk 1 above)
Comment (SamCarlberg) [reply source/docs/software/telemetry/robot-telemetry-with-annotations.rst:4]: I agree. Is there a standard format we should follow (such as prepending "New for 2025" to the title)?

File: source/docs/software/telemetry/robot-telemetry-with-annotations.rst [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+          }
+        }
+
+
+Logging Third-Party Data
+------------------------
+
+A large part of the Java ecosystem is its access to third-party libraries that provide extra functionality, such as vendor libraries for interacting with custom hardware like motor controllers and sensors. If those vendors do not use logging annotations in their libraries, then you can still log them by writing a custom logger class! These classes are actually what the WPILib annotations will generate; you just write a custom one because third-party code cannot be annotated.
+
+Custom loggers can be declared in any package, and only need to have the ``@CustomLoggerFor`` annotation present to be able to be detected and used. They must extend from ``ClassSpecificLogger`` (`Javadoc <https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/epilogue/logging/ClassSpecificLogger.html>`__) and must have a public no-argument construtor - failing to declare one will result in a compilation error - and must pass the type they log to the superclass constructor.
```
Comment (SamCarlberg) [inline source/docs/software/telemetry/robot-telemetry-with-annotations.rst:262]: I don't think we publish javadocs for latest/bleeding edge builds so this link will be broken until there's a release

Comment (sciencewhiz) [reply source/docs/software/telemetry/robot-telemetry-with-annotations.rst:262]:
```suggestion
Custom loggers can be declared in any package, and only need to have the ``@CustomLoggerFor`` annotation present to be able to be detected and used. They must extend from ``ClassSpecificLogger`` (`Javadoc <https://github.wpilib.org/allwpilib/docs/development/java/edu/wpi/first/epilogue/logging/ClassSpecificLogger.html>`__) and must have a public no-argument construtor - failing to declare one will result in a compilation error - and must pass the type they log to the superclass constructor.
```

File: source/docs/software/telemetry/robot-telemetry-with-annotations.rst (hunk 1 above)
Comment (sciencewhiz) [reply source/docs/software/telemetry/robot-telemetry-with-annotations.rst:4]: Not really, but here's one example. https://docs.wpilib.org/en/stable/docs/software/networktables/nt4-migration-guide.html#migrating-from-networktables-3-0-to-networktables-4-0

**Pull request merged**